Raspi/Test/Pipfile

```diff
@@ -1,13 +1,16 @@
 [[source]]
-url = "https://pypi.python.org/simple"
-verify_ssl = true
 name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
 
 [packages]
 paho-mqtt = "*"
-"urllib3" = "*"
-
-[dev-packages]
+scipy = "*"
```

Comment:
What do we use `scipy` for?

Reply:
Oops I think I must have installed it by accident